python/lsst/rubintv/production/oneOffProcessing.py

```diff
@@ -166,14 +166,34 @@ def calcPsfAndWrite(self, exp: Exposure, dayObs: int, seqNum: int) -> None:
             self.log.info(f"Wrote measured PSF for {dayObs}-{seqNum} det={exp.detector.getId()}: {fwhm:.3f}")
         except Exception as e:
             self.log.error(f"Failed to calculate PSF for {dayObs}-{seqNum} det={exp.detector.getId()}: {e}")
-            if self.doRaise:
-                raise
```

Comment:
Did you mean to remove these lines? If so, it would be helpful to put them in a separate commit.

Reply:
Yeah, the whole `doRaise` pattern isn't really being adhered to in this new module tbh, so I think I'll take this as the poke I needed to make it adhere to that, thanks.